infrastructure/keycloak/realm-export.json

```diff
@@ -508,33 +508,6 @@
       "notBefore": 0,
       "groups": []
     },
-    {
-      "id": "3c0bce2e-ba21-42df-9e66-9e6f15431b6b",
-      "createdTimestamp": 1621530527270,
-      "username": "service-account-ai-reviewer-ui",
-      "enabled": true,
-      "totp": false,
-      "emailVerified": false,
-      "serviceAccountClientId": "ai-reviewer-ui",
-      "disableableCredentialTypes": [],
-      "requiredActions": [],
-      "realmRoles": [
-        "uma_authorization",
-        "offline_access"
-      ],
-      "clientRoles": {
-        "ai-reviewer-api": [
-          "ai-reviewer-api-admin",
-          "ai-reviewer-api-client"
-        ],
-        "account": [
-          "view-profile",
-          "manage-account"
-        ]
-      },
-      "notBefore": 0,
-      "groups": []
-    },
```

Comment:
I just want to confirm the removal of this service account.  The ai-reviewer-ui client was added by Adam and used and configured in DEV.  This commit is removing this client id.

If I fix the PORT=3002 issue in the .env.template file, I see this when I launch the application:
![image](https://user-images.githubusercontent.com/55215368/119735302-f8fd7580-be30-11eb-8a59-f9f7c4c7fc60.png)

Notice the url is using client-id=ai-reviewer-ui as a parameter, but it's being removed here and so doesn't exist.

Remember to **docker-compose down** to remove cached containers and then **docker-compose up -d --build** to rebuild them.

Reply:
okay I see whats happened, strange that it works for me though. So I had to make the route public for ai-reviewer-ui in order to get it to authenticate (as it is in efiling), however it looks like this action removes service accounts for that client. I'm on it.

Reply:
Yeah, but I think though that making the ai-reviewer-ui public is the right call.  
We should get Taylor and Adam to chime in.

I read this in the documentation:
https://www.keycloak.org/docs/latest/securing_apps/#_javascript_adapter
_One important thing to note about using client-side applications is that the client has to be a public client as there is no secure way to store client credentials in a client-side application._
(per https://stackoverflow.com/questions/62298549/keycloak-client-secret-not-provided-in-request)

Reply:
Thats the documentation I saw as well. Also efiling-demo is configured as public. I'm just waiting for Adam to get back to me on this.

Reply:
Got the go ahead from Adam its just going to stay public like efiling

Reply:
Okay, cool.   I still can't get the app the run locally, but if it works for you then it must be my env.
Awesome work!